src/sync.js

```diff
@@ -511,6 +511,12 @@ SyncApi.prototype.sync = function() {
         checkLazyLoadStatus(); // advance to the next stage
     }
 
+    function createDefaultFilter() {
```

Comment:
Maybe call this `build...` to differentiate from `create` which is a network call for filters?